blender/LilySurfaceScrapper/preferences.py

```diff
@@ -21,31 +21,91 @@
 # This file is part of LilySurfaceScrapper, a Blender add-on to import materials
 # from a single URL
 
+import os.path
 import bpy
+import sys
+import os
 
 addon_idname = __package__.split(".")[0]
 
 # -----------------------------------------------------------------------------
 
+def modules_path():
+    # Copied and adapted from https://github.com/jesterKing/import_3dm/blob/9f96e644e40edd571829cbaf777d6bda63dbb5db/import_3dm/read3dm.py
+    # set up addons/modules under the user
+    # script path. Here we'll install the
+    # dependencies
+    modulespath = os.path.normpath(
+        os.path.join(
+            bpy.utils.script_path_user(),
+            "addons",
+            "modules"
+        )
+    )
+    if not os.path.exists(modulespath):
+        os.makedirs(modulespath)
+
+    # set user modules path at beginning of paths for earlier hit
+    if sys.path[1] != modulespath:
+        sys.path.insert(1, modulespath)
+
+    return modulespath
+
+modules_path()
```

Comment:
Isn't this a leftover of a test?

Reply:
I copy and pasted this from the source I mentioned :)

I think line 54 is there to ensure that `modulespath` is added to the start of `sys.path` for performance reasons.

Currently, if the lxml import test passes in `ensureLxmlInstalled`, `modules_path` is never called and this side-effects is never applied.

I think that we can safely remove line 54 and lines 49-50 and not mangle the default order of the `sys.path` array. I pushed another commit!